src/release/program-github-promote-release.ts

```diff
@@ -0,0 +1,68 @@
+import { Octokit } from '@octokit/rest'
```

Comment:
This file is completely copied from Integreat.

Reply:
Yes I mentioned this in the description